library-backend/library-app/src/main/java/org/internship/library/app/persistence/LibraryJpaConfig.java

```diff
@@ -0,0 +1,32 @@
+package org.internship.library.app.persistence;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+import org.internship.library.api.Book;
+import org.internship.library.app.persistence.entity.BookEntity;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@EnableAutoConfiguration
+public class LibraryJpaConfig implements InitializingBean {
```

Comment:
This should not be named with `JPA` inside. Switch to `LibraryPersistenceConfig`.